api/HTMLInputElement.json

```diff
@@ -543,13 +534,15 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "3.6"
+              "version_added": "3"
             },
             "firefox_android": {
               "version_added": "4"
             },
             "ie": {
-              "version_added": "5.5"
+              "version_added": "10",
+              "partial_implementation": true,
+              "notes": "This property is read-only."
```

Comment:
This and other notes aren't being added/removed in this PR, it's just entries that appear to be changed in the diff due to other entries being added. I reviewed the actual differences using https://github.com/mdn/browser-compat-data/pull/6862